src/frontend/src/main/SearchField/index.jsx

```diff
@@ -90,13 +90,26 @@ export default function SearchField({ onLoading, setTicketsData, setRequestBody
     setRequestBody(body);
     onLoading(true);
     setTicketsData([]);
-    const dataStream = await eventSourceQuery('searchTickets', JSON.stringify(body));
-    onLoading(false);
-    for await (const chunk of dataStream) {
-      if (chunk) {
-        setTicketsData((prevTickets) => [...prevTickets, chunk]);
-      }
+    function onClose() {
```

Comment:
Please, change name from onClose -> handleClose
also, if this "onLoading" - use useState, rename. If it is method - handleLoading